notebook/debugging/data_similarity.ipynb

```diff
@@ -0,0 +1,1306 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.hist(nonzero_distances, bins=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Distribution looks bimodal, probably due to different platforms having different distances from each other?"
```

Comment:
That's a good idea, but I can't think of a clean way to do platforms since the distances correspond to pairs of samples. Also there's 50 million entries and my computer started swapping when I tried to do something similar with the distances at the end